schema/config-linux.json

```diff
@@ -163,10 +163,17 @@
                             },
                             "blkioWeightDevice": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/blockIO/blkioWeightDevice",
-                                "type": "array",
-                                "items": {
-                                    "$ref": "defs-linux.json#/definitions/blockIODeviceWeightPointer"
-                                }
+                                "oneOf": [
+                                    {
+                                        "type": "array",
+                                        "items": {
+                                            "$ref": "defs-linux.json#/definitions/blockIODeviceWeight"
+                                        }
+                                    },
+                                    {
+                                        "type": "null"
```

Comment:
We probably don't want the `null` alternative since null alternatives for arrays were [just rejected][1] in #555.

[1]: https://github.com/opencontainers/runtime-spec/pull/555#issuecomment-272020515